release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,49 +1,27 @@
-    - [Reports [Before Freeze]](#Reports-Before-Freeze)
-    - [Issue Categorization](#Issue-Categorization)
-  - [Code Freeze](#Code-Freeze)
-    - [Sample Searches [Freeze]](#Sample-Searches-Freeze)
-    - [Reports [Freeze]](#Reports-Freeze)
-  - [Code Thaw](#Code-Thaw)
-    - [Sample Searches [Thaw]](#Sample-Searches-Thaw)
-    - [Reports [Thaw]](#Reports-Thaw)
-
-## Summary
-The role of the bug triage team is to make sure that Pull Requests (PRs) and Github Issues which affect the release are dealt with in a timely fashion.
-
-Primary tasks include:
-- Listing the status of open issues / PRs
-- Communicating with relevant assignees / owners / sig-leads of issues/PRs to get status
-- Updating issues/PRs, clarifying situations, enabling next level decision making
-- Presenting summary reports at release meetings
+# Bug Triage Role Handbook
 
-There are four relevant periods where the workload changes:
+## Overview
+
+The bug triage team is responsible to make sure that Issues and Pull Requests (PRs) which are targeted for the ongoing release cycle are dealt with in a timely fashion.
+
+The role has been interchangeably called "Bug Triage" and "Issue Triage", but the general term "issue" can be conflated with the specific GitHub usage of `is:issue` artifact type. It is important to consider both `is:issue` and `is:pr` GitHub artifacts when triaging bugs.
 
-1. Early Release: from day 1 up to about a week before Code Freeze. *Duration: ~ 7-8 weeks*
-2. Code Freeze is Coming: ~2 weeks before Code Freeze. *Duration: ~2 weeks*
-3. Code Freeze: Code Freeze & Burndown, beta releases, until Code Thaw. *Duration: ~2 weeks*
-4. Code Thaw: Last two weeks of the cycle. *Duration: ~2 weeks*
+## Responsibilities
 
-Secondarily, you will be helping improve automation around tracking issues against the current milestone during the Early Release phase.
+The bug triage role has the following responsibilities:
 
-The role has been interchangeably called "Bug Triage" and "Issue Triage", but the general term "issue" can be conflated with the specific GitHub usage of `is:issue` artifact type.  It is important to consider both `is:issue` and `is:pr` GitHub artifacts when triaging bugs.
+- Listing open issues and PRs targeted for the ongoing release cycle
+- Communicating with relevant assignees, owners, and SIG leads of issues/PRs to get the status
+- Updating issues/PRs, clarifying situations, enabling next level decision making
+- Presenting summary reports at release team and burndown meetings
+- Maintaining the automation around tracking issues/PRs against the current milestone
+
+There are four relevant periods where the workload changes:
+
+1. Early Release: from week 0 up to mid-release cycle (week 6). *Duration: ~6-7 weeks*
+2. Mid release: from week 6 up to the code freeze (around week 9). *Duration: ~3-4 weeks*
+3. Code Freeze: from around week 9 up to the code thaw (week 11). *Duration: ~2 weeks*
```

Comment:
Do we have a definition somewhere how long each stage lasts (i.e. when the code freeze starts)? For example, is week 9 a good guess here?

Reply:
9's a reasonable guess.

There is https://git.k8s.io/community/contributors/devel/sig-release/release.md for a reference.